executors/cpp/number_fmt.cpp

```diff
@@ -91,35 +305,59 @@ const string test_numfmt(json_object *json_in) {
 
   // Get the numeric value
   double input_double = std::stod(input_string);
-  std:: cout<< "num_double = " << input_double << std::endl;
 
-  // TEMPORARY - just return the input.
-  test_result = input_string;
+  LocalizedNumberFormatter nf;
+  if (notation_string == "scientific") {
+    notation_setting = Notation::scientific();
+  }
 
-  // TODO!!!
-  //  SimpleNumberFormatter snf = SimpleNumberFormatter::forLocale(displayLocale, status);
```

Comment:
This approach is fine but I would also have been okay with an approach that constructs a NumberFormatter from the skeleton string. If we don't use the skeleton string, I think we should remove it from the input data and keep it, at best, in some sort of annotation.

Reply:
Now working on using the skeleton where it exists.